chargeOclock/Source/Other/NetworkAPI.swift

```diff
@@ -16,6 +16,9 @@ extension NetworkAPI {
 		performGetRequest(to: "clients", completion: completion)
 	}
 
+	func removeClient(with id: Int, completion: @escaping NetworkCompletion) {
```

Comment:
please rename to `deleteClient` (same argument as in one of the previous comments)